nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/src/main/java/org/apache/nifi/web/security/saml/impl/StandardSAMLStateManager.java

```diff
@@ -140,4 +140,7 @@ public String getJwt(final String requestIdentifier) {
         }
     }
 
+    public void setJwtService(JwtService jwtService) {
+        this.jwtService = jwtService;
+    }
```

Comment:
Is this change necessary?  It looks like the JwtService is provided through a constructor argument as part of the existing Spring configuration.